app/src/main/res/values/strings.xml

```diff
@@ -1032,15 +1032,17 @@
     <string name="force_update_needed_message">You must update your Concordium Legacy Wallet to continue.\n\nIf you don’t have a backup, Concordium strongly recommends making one before updating.</string>
     <string name="force_update_needed_update_now">Update now</string>
     <string name="force_update_needed_backup">Make backup</string>
-    <string name="force_update_warning_title">Update ready</string>
-    <string name="force_update_warning_message">There is a new update for the Concordium Legacy Wallet.\n\nIf you don’t have a backup, Concordium strongly recommends making one before updating.</string>
+
+    <string name="force_update_warning_title">New wallet app available</string>
+    <string name="force_update_warning_message">Going forward the Concordium Legacy Wallet will not receive any new features. We therefor recommend that you install the new Concordium Wallet for Mobile, to be able to use all the latest Concordium features. You will still be able to access your existing accounts and identities in the Concordium Legacy Wallet, but be sure to keep a backup of your private keys.</string>
```

Comment:
```suggestion
    <string name="force_update_warning_message">Going forward the Concordium Legacy Wallet will not receive any new features. We therefore recommend that you install the new Concordium Wallet for Mobile, to be able to use all the latest Concordium features. You will still be able to access your existing accounts and identities in the Concordium Legacy Wallet, but be sure to keep a backup of your private keys.</string>
```

Reply:
The text change is already addressed and will be in the next build.
Regarding the other changes, some of the changes were needed to build and run the app like for example the  room library version, than that library for example has minimum required Gradle version which requires Kotlin and other library updates. Those changes were small and enabled me to test and run the app. Then I run in to a problem that I could not build the release  APK file and had to make other changes to fix that.  Those changes are the minimum I had to do.

Reply:
The text change is already addressed and will be in the next build.
Regarding the other changes, some of the changes were needed to build and run the app like for example the room library version, then that library for example has minimum required Gradle version which requires Kotlin and other library updates. Those changes were small and enabled me to test and run the app. Then I run in to a problem that I could not build the release APK file and had to make other changes to fix that. Those changes are the minimum I had to do.